lib/azure/armrest/armrest_service.rb

```diff
@@ -230,12 +232,12 @@ def subscriptions
       # subscription ID that was provided in the constructor if none is
       # specified.
       #
-      def subscription_info(subscription_id = armrest_configuration.subscription_id)
+      def subscription_info(subscription_id = configuration.subscription_id)
         url = url_with_api_version(
-          armrest_configuration.api_version,
+          configuration.api_version,
           @base_url,
           'subscriptions',
-          armrest_configuration.subscription_id
+          configuration.subscription_id
```

Comment:
Yep, good catch by rubocop. There were a couple other minor things it found, and I realized that we had documented and tested one StorageAccount method incorrectly as well.

I pushed up those changes.